modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -143,6 +143,9 @@ trait ObserveCommand {
   def setTimeout[F[_]: Sync](t: Time): F[Unit] =
     Sync[F].delay {
       cs.map(_.getApplySender).map(_.setTimeout(t.toMilliseconds.toLong, MILLISECONDS))
+    } *>
+    Sync[F].delay {
```

Comment:
@jluhrs I traced how the timeouts on CA are set and I think we are missing this one. It seems this is the core reason for this bug. I'll try to test it on instrument

Reply:
Yes! This is the core reason. But the new line must replace the old one. This
```
    Sync[F].delay {
      cs.map(_.getApplySender).map(_.setTimeout(t.toMilliseconds.toLong, MILLISECONDS))
    } *>
```
must go away.